src/language/LanguageProvider.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { createContext, useContext, useEffect } from "react";
```

Comment:
```suggestion
import React, { createContext, useContext } from "react";
```
You dont use this one